src/utils/helpers/filterTodos.tsx

```diff
@@ -0,0 +1,19 @@
+import { Todo } from '../../types/Todo';
+
+export const filterTodos = (todos: Todo[], filterBy: string) => {
+  if (todos) {
```

Comment:
But this parameter is required, why should we check if it passed? Let typescript do it for you